app-containers/kubectl/autobuild/build

```diff
@@ -0,0 +1,13 @@
+abinfo "Build kubectl ..."
+make kubectl
+
+abinfo "Install kubectl ..."
+install -Dvm755 ./_output/bin/kubectl -t "$PKGDIR"/usr/bin
```

Comment:
Use absolute path to make your code more readable, i.e., `"$SRCDIR"/_output/bin/kubectl`.